logstash-core/lib/logstash-core/version.rb

```diff
@@ -5,4 +5,4 @@
 # Note to authors: this should not include dashes because 'gem' barfs if
 # you include a dash in the version string.
 
-LOGSTASH_CORE_VERSION = "6.0.0-alpha1"
+LOGSTASH_CORE_VERSION = "6.0.0.alpha1"
```

Comment:
@ph this is gonna mess up the unified release. We'll have different package names compared to other products like `logstash-6.0.0.alpha1.zip` instead of `logstash-6.0.0-alpha1.zip`